gstd/src/msg/async.rs

```diff
@@ -22,9 +22,9 @@ use crate::{
     async_runtime::{signals, ReplyPoll},
```

Comment:
Now we may rename this module to `msg::future` and correct docs that there is no async fns, but their return types